docs/docsite/rst/conf.py

```diff
@@ -386,3 +392,13 @@
 ]
 linkcheck_workers = 25
 # linkcheck_anchors = False
+
+# Generate redirects for pages that have been restructured or moved
+# if EXTRA_TAGS="-t redirects" is passed with the make invocation
```

Comment:
Is this for Read The Docs only? It's possible to detect that env automatically.

Reply:
Ah, right. I forgot about that `READTHEDOCS = true`. Nice.

Reply:
Ah, I see. It can still be integrated for future, I suppose. In the past, I would auto-add tags based on that var so it would work with an explicit tag + an implicitly added one.

Is https://github.com/ansible/ansible-documentation/blob/7ff911e/.readthedocs.yaml disabled, then?

Reply:
No, that config is used for the core docs builds: https://ansible.readthedocs.io/projects/ansible-core/devel/

As it happens I sent you a maintainer invite for that project but it expired. Do you want me to resend it?